test-framework/kubernetes-client/src/main/java/io/quarkus/test/kubernetes/client/AbstractKubernetesFixturesTestResource.java

```diff
@@ -0,0 +1,55 @@
+package io.quarkus.test.kubernetes.client;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import io.fabric8.kubernetes.api.model.HasMetadata;
+
+public abstract class AbstractKubernetesFixturesTestResource extends
```

Comment:
I don't understand the term fixtures. Care to elaborate?

Reply:
See https://en.wikipedia.org/wiki/Test_fixture